docs/blocks/blocks.md

```diff
@@ -8,18 +8,30 @@ Creating a Block
 
 ### Basic Blocks
 
-For simple blocks, which need no special functionality (think cobblestone, wooden planks, etc.), a custom class is not necessary. You can create a block by instantiating the `Block` class with a `Block.Properties` object. This `Block.Properties` object can be made using `Block.Properties#create` and it can be customised by calling its methods. For instance:
+If the block you are creating has no special functionality (think cobblestone, wooden planks), it is not necessary to create a new class for your block. You can simply instantiate the `Block` class. When doing this you will need a `Block.Properties` to pass to the constructor. This is how we set the properties of a block in 1.15+. Here's an exmaple:
 
-- `hardnessAndResistance` - The hardness controls the time it takes to break the block. It is an arbitrary value. For reference, stone has a hardness of 1.5, and dirt 0.5. If the block should be unbreakable a hardness of -1.0 should be used, see the definition of `bedrock` as an example. The resistance controls the explosion resistance of the block. For reference, stone has a resistance of 6.0, and dirt 0.5.
-- `sound` - Controls the sound the block makes when it is punched, broken, or placed. Requires a `SoundType` argument, see the [sounds][] page for more details.
-- `lightValue` - Controls the light emission of the block. Takes a value from zero to fifteen.
-- `slipperiness` - Controls how slippery the block is. For reference, ice has a slipperiness of 0.98.
+```java
+new Block(Block.Properties.from(Blocks.COBBLESTONE).lightValue(15)).setRegistryName("mymod:myblock");
```

Comment:
This is not completely valid. You need to specify the registry method if you plan on defining it this way. `DeferredRegister` will throw an ISE if you use this as your block.

The registry name should standardize to `examplemod` as that is the name in relation to the mdk (minor nitpick).